kaldb/src/test/java/com/slack/kaldb/chunk/ReadWriteChunkImplTest.java

```diff
@@ -424,24 +421,24 @@ public void testSnapshotToS3UsingChunkApi() throws Exception {
       chunk.close();
       chunk.cleanup();
 
-      // Download data from S3, create a new chunk and query the data in that chunk.
+      // TODO: Download data from S3, create a new chunk and query the data in that chunk.
       // Download files from S3 to local FS.
-      File newLocalFolderPath = localDownloadFolder.newFolder();
-      String[] s3Files =
-          copyFromS3(bucket, "", s3BlobFs, Paths.get(newLocalFolderPath.getAbsolutePath()));
-      assertThat(FileUtils.listFiles(newLocalFolderPath, null, true).size())
-          .isEqualTo(s3Files.length);
-
-      ReadOnlyChunkImpl<LogMessage> readOnlyChunk =
-          new ReadOnlyChunkImpl<>(
-              newLocalFolderPath.getAbsoluteFile().toPath(),
-              new ChunkInfo("testDataSet2", 0),
-              registry);
-      SearchResult<LogMessage> newChunkResults = readOnlyChunk.query(searchQuery);
-      assertThat(newChunkResults.hits.size()).isEqualTo(1);
-      readOnlyChunk.close();
-      assertThat(FileUtils.listFiles(newLocalFolderPath, null, true).size())
-          .isEqualTo(s3Files.length);
+      //      File newLocalFolderPath = localDownloadFolder.newFolder();
```

Comment:
Is there a reason to comment out this part of the unit test?

Reply:
Ah yes, I've now removed this. This test used the previous stub of `ReadOnlyChunkImpl`, which allowed instantiation of a `ReadOnlyChunkImpl` against a directory on disk. This is no longer possible as the `ReadOnlyChunkImpl` is now wired against ZK events.

All of the tests for verifying the `ReadOnlyChunkImpl` can hydrate / load / query assets from S3 were moved into the `ReadOnlyChunkImplTest` as part of #124 - (https://github.com/slackhq/kaldb/blob/6f578681ae137b90654ee10cdd47745fbc8e1d61/kaldb/src/test/java/com/slack/kaldb/chunk/ReadOnlyChunkImplTest.java)